### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by brdesai**
Adding instructions on how to unassign a Customer Key DEP to a mailbox.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -163,6 +163,13 @@ The output from this cmdlet includes:
 
   - **Rolling:** A key roll is in progress. If the key for the geo is rolling, you'll also be shown information on what percentage of sites have completed the key roll operation so that you can monitor progress.
 
+## Unassign DEP from a mailbox
+
+You can unassign a Customer Key DEP from a mailbox by assigning $NULL DEP to the mailbox. Doing so will unassign the currently assigned Customer Key DEP and reencrypt the mailbox using the default Microsoft managed keys DEP. You can not unassign the Microsoft managed keys DEP. You can reassign another Customer Key DEP to the same mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
You can unassign a Customer Key DEP from a mailbox by assigning $NULL DEP to the mailbox. Doing so will unassign the currently assigned Customer Key DEP and re-encrypt the mailbox using the default Microsoft managed keys DEP. You can not unassign the Microsoft managed keys DEP. You can reassign another Customer Key DEP to the same mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by brdesai** [comment]
Thanks @JohanFreelancer9  for the review. Who is going to push the changes to Public?

File: microsoft-365/compliance/customer-key-manage.md (hunk 1 above)
Comment (KCCross) [reply microsoft-365/compliance/customer-key-manage.md]: Needs a bit of an Acrolinx check. I will work on this one in the private repo tonight.

**Review by brdesai — COMMENTED** [review]
accepted the suggested change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by KCCross** [comment]
Ready for publication.

**Pull request merged**